.github/workflows/release.yml

```diff
@@ -0,0 +1,81 @@
+# Copyright KubeArchive Authors
+# SPDX-License-Identifier: Apache-2.0
+---
+name: Release
+on:
+  workflow_dispatch:
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+      - run: echo "Branch ${{ github.ref }} of repository ${{ github.repository }}."
+      - name: Check out repository code
+        uses: actions/checkout@v4
+      - name: Setup Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: '1.22'
+      - name: Install tools
+        run: |
+          go install k8s.io/release/cmd/release-notes@latest
+          go install github.com/mikefarah/yq/v4@latest
+      - name: Generate notes
+        run: |
+          export ORG=kubearchive
```

Comment:
Would it be possible to make this a script in the repo rather then embed the code inside the YAML file? It would be easier to test and maintain that way (I spotted one error already).

Reply:
I'm working on it. I think it will help with testing it locally, which I'm also currently trying.